source/go-perun/multi_ledger_tutorial/index.rst

```diff
@@ -0,0 +1,160 @@
+   :language: go
+   :lines: 110-121
+
+Finally, we can create our Payment client which gets the Perun client, and then start the handler of the Perun client to
+handle channel and update proposals.
+
+.. literalinclude:: ../../perun-examples/multiledger-channel/client/client.go
+   :caption: `👇 This code on GitHub. <https://github.com/hyperledger-labs/go-perun/blob/e831f7ec3d1fee283cc66a3035969a2c7b9aad71/channel/multi/funder.go#L26>`__
+   :language: go
+   :lines: 123-130
+
+Channel opening
+...............
+
+The channel opening is again very similar to the payment channel tutorial, the only difference is visible
+when setting the allocations of the channel proposal.
+There, we now have two assets (the ERC20 tokens) for the two chains for which we need to define the balances for Alice
+and Bob.
+So, the ``OpenChannel`` method takes two balance arrays for the two chains as an input.
+
+.. literalinclude:: ../../perun-examples/multiledger-channel/client/client.go
+   :caption: `👇 This code on GitHub. <https://github.com/hyperledger-labs/go-perun/blob/e831f7ec3d1fee283cc66a3035969a2c7b9aad71/channel/multi/funder.go#L26>`__
+   :language: go
+   :lines: 135-136
+
+When looking at the ``main.go``, we can see how we would call this function.
+
+.. literalinclude:: ../../perun-examples/multiledger-channel/main.go
+   :caption: `👇 This code on GitHub. <https://github.com/hyperledger-labs/go-perun/blob/e831f7ec3d1fee283cc66a3035969a2c7b9aad71/channel/multi/funder.go#L26>`__
+   :language: go
+   :lines: 77-81
+
+
+Going back to the ``OpenChannel`` method itself, we can take a look at how we set the allocation for the proposal.
+First, we create a new allocation with ``NewAllocation`` which takes the number of participants and the assets of the
+channel as an input.
+Then, we can set the balances for Alice and Bob for the corresponding asset using the two given balance arrays.
+
+.. literalinclude:: ../../perun-examples/multiledger-channel/client/client.go
+   :caption: `👇 This code on GitHub. <https://github.com/hyperledger-labs/go-perun/blob/e831f7ec3d1fee283cc66a3035969a2c7b9aad71/channel/multi/funder.go#L26>`__
+   :language: go
+   :lines: 143-154
+
+
+After Alice has sent the channel proposal, Bob will automatically accept the proposal after checking its correctness
+(see :ref:`handle proposals <payment_client_handle_proposals>` for more information).
+
+Channel Update
+..............
+
+Using the established channel, Alice and Bob can now perform off-chain updates, meaning they can sent each other nearly
+instant fee-less payments with the two ERC20 tokens that live on different chains.
+For this, we created the ``SendPayment`` method which sends the given amount to the other party. We use the chain index
+to specify which asset on which chain we want to use (the chain indices are defined in ``main.go``).
```

Comment:
This is not a good example for a cross-chain application. If this was just about payments, we wouldn't need a multi-chain channel. Please modify the example to do a swap.

Reply:
Done.